lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
According to [this test](https://github.com/tc39/test262/blob/master/test/language/statements/for-of/const-fresh-binding-per-iteration-for-of.js) each iteration is supposed to create a fresh binding, so it looks like a limitation in V8's sloppy mode. Behavior in strict mode is correct.